app/src/renderer/main.js

```diff
@@ -46,23 +45,9 @@ Vue.use(Router)
 Vue.use(Vuelidate)
 
 async function main () {
-  let nodeIP = getQueryParameter('node')
-  if (!nodeIP) {
-    console.log('Did not receive a node to connect to')
-    return
-  }
-
-  let relayPort = getQueryParameter('relay_port')
-  console.log('Expecting relay-server on port:', relayPort)
-  console.log('Connecting to node:', nodeIP)
-  const node = Node(nodeIP, relayPort)
-
-  node.lcdConnected()
-    .then(connected => {
-      if (connected) {
-        axios.get(`http://localhost:${relayPort}/startsuccess`)
-      }
-    })
+  let lcdPort = getQueryParameter('lcd_port')
+  console.log('Expecting lcd-server on port:', lcdPort)
+  const node = Node('localhost', lcdPort)
```

Comment:
isn't this defaulting to a localhost now instead of the actual node we want to connect to?

Reply:
This is confusing I will change this.